src/picongpu/include/fields/background/templates/TWTS/BField.hpp

```diff
@@ -41,6 +41,14 @@ class BField
 public:
     typedef float_X float_T;
     
+    enum PolarizationType
+    {
+        /* The linear polarization of the TWTS laser is defined
+         * relative to the plane of the pulse front tilt. */
+        NORMAL_TO_TILTPLANE = 1u, /* use Ex-fields in TWTS laser internal coordinate system */
```

Comment:
from the naming conventions, we usually [define polarization like this](https://github.com/ComputationalRadiationPhysics/picongpu/blob/1f2ffd2ecf0aba3b372d2a641f0758b5de6814d3/src/picongpu/include/simulation_defines/param/laserConfig.param#L78-L84)

can we use the same namings here?